docs/agents/labjack.rst

```diff
@@ -32,10 +32,37 @@ available arguments::
          'arguments':[
            ['--ip-address', '10.10.10.150'],
            ['--num-channels', '13'],
+           ['--function-file', 'labjack_functions.yml'],
            ['--mode', 'acq'],
            ]},
 
-You should assign your LabJack a static IP, you'll need to know that here.
+You should assign your LabJack a static IP, you'll need to know that here. 
+The 'functions-file' argument specifies the labjack configuration file, 
```

Comment:
'functions-file' differs from the argument 'function-file', please make that consistent.

Reply:
done